README.md

```diff
@@ -1,151 +1,34 @@
-LaunchDarkly SDK for PHP
-===========================
+# LaunchDarkly Server-side SDK for PHP
 
-[![Circle CI](https://circleci.com/gh/launchdarkly/php-client.svg?style=svg)](https://circleci.com/gh/launchdarkly/php-client)
+[![Circle CI](https://img.shields.io/circleci/project/launchdarkly/php-server-sdk.png)](https://circleci.com/gh/launchdarkly/php-server-sdk)
 
-Requirements
-------------
-1. PHP 5.5 or higher. 
+## LaunchDarkly overview
 
-Quick setup
------------
+[LaunchDarkly](https://www.launchdarkly.com) is a feature management platform that serves over 100 billion feature flags daily to help teams build better software, faster. [Get started](https://docs.launchdarkly.com/docs/getting-started) using LaunchDarkly today!
+ 
+[![Twitter Follow](https://img.shields.io/twitter/follow/launchdarkly.svg?style=social&label=Follow&maxAge=2592000)](https://twitter.com/intent/follow?screen_name=launchdarkly)
 
-1. Install the PHP SDK and monolog for logging with [Composer](https://getcomposer.org/)
+## Supported PHP versions
 
-        php composer.phar require launchdarkly/launchdarkly-php
+This version of the LaunchDarkly SDK is compatible with PHP 5.5 and higher.
 
-1. After installing, require Composer's autoloader:
+## Getting started
 
-		require 'vendor/autoload.php';
+Refer to the [SDK reference guide](https://docs.launchdarkly.com/docs/php-sdk-reference) for instructions on getting started with using the SDK.
```

Comment:
You may as well include the section anchor: https://docs.launchdarkly.com/docs/php-sdk-reference#section-getting-started